src/components/calcite-radio-button-group/calcite-radio-button-group.tsx

```diff
@@ -37,16 +37,7 @@ export class CalciteRadioButtonGroup {
   }
 
   /** The layout direction of the radio buttons in a group. */
-  @Prop({ mutable: true, reflect: true }) layout: "horizontal" | "vertical" = "horizontal";
-
-  @Watch("layout")
-  validateLayout(newLayout: string) {
-    const layouts = ["horizontal", "vertical"];
-    if (!layouts.includes(newLayout)) {
-      this.layout = "horizontal";
-      this.passPropsToRadioButtons();
-    }
-  }
```

Comment:
Good call on removing this whole thing, including the call to `passPropsToRadioButtons` which has no effect because radio-buttons don't receive the layout prop.